prac_08/squaring.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+Kivy GUI program to square a number
+Lindsay Ward, IT@JCU
```

Comment:
It would be good if you mention a little bit about the program in your docstring